### PR TITLE
enh(UI): adjust icon size

### PR DESCRIPTION
**Pull request by ponchoh**
## Description

set a size to prevent overblown size if image is bigger

## Type of change

- [x] Patch fixing an issue (non-breaking change)

## Target serie

- [ ] 20.04.x
- [x] 20.10.x
- [x] 21.04.x
- [x] 21.10.x (master)

## How this pull request can be tested ?

Better view

image

VS

image

## Checklist

- [x] I have followed the **coding style guidelines** provided by Centreon
- [x] I have **rebased** my development branch on the base branch (develop).

### CODE REVIEW

**Review by kduret — DISMISSED** [review]

**Review by jeremyjaouen — DISMISSED** [review]

File: Jenkinsfile [hunk 1]
```diff
@@ -125,7 +125,7 @@ def checkoutCentreonBuild(buildBranch) {
 /*
 ** Pipeline code.
 */
-stage('Source') {
+stage('Deliver sources') {
```
Comment (sc979) [inline Jenkinsfile:128]: Jenkinsfile modifications are out of the PR description scope.
Could you rebase your work ?

**Review by sc979 — CHANGES_REQUESTED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (zguennoune02) [reply Jenkinsfile:128]: I pushed a commit in order to avoid rpm builded from CI branches to go on unstable repo.

**Review by sc979 — DISMISSED** [review]
Branch rebased on October, 11

**Review by kduret — CHANGES_REQUESTED** [review]

File: www/Themes/Centreon-2/style.css [hunk 2]
```diff
@@ -1159,6 +1159,11 @@ div.menuLeft {
     margin: 2px 10px 2px 0px;
 }
 
+#esi_icon_image {
```
[the remark this answers is not in the document]
Comment (ponchoh) [reply www/Themes/Centreon-2/style.css]: done 1302ca01fd0af852ce8c223d4ef77048e2013a87

**Review by kduret — DISMISSED** [review]

**Review by sc979 — DISMISSED** [review]

**Comment by ponchoh** [comment]
MON-12791

**Review by Thebarda — DISMISSED** [review]

**Review by jdelpierre — DISMISSED** [review]

**Pull request closed without merging**